index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather App</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <h1 class="heading">Weather App</h1>
+        
+        <form id="weatherForm">
+          <label for="cityName" class="label">City Name:</label>
+          <input type="text" id="cityName" name="cityName" required class="input-text">
+          <button type="submit" class="submit-btn">Get Weather</button>
+        </form>
+        <div id="weatherInfo" class="weather-info"></div>
+    </div>
+
+    <script src="weatherService.js"></script>
```

Comment:
the script is now in src dir, this wont work, it needs to be src/weatherService.js